examples/crm/src/dataGenerator/tasks.ts

```diff
@@ -3,34 +3,7 @@ import { lorem, random } from 'faker/locale/en_US';
 import { Task } from '../types';
 import { Db } from './types';
 import { randomDate } from './utils';
-
-export const type: Task['type'][] = [
```

Comment:
Nope, this should stay in the data generator. It controls the likelihood of each option.